src/server_manager/web_app/ui_components/app-root.js

```diff
@@ -564,26 +508,37 @@ export class AppRoot extends mixinBehaviors
     this.currentPage = 'intro';
   }
 
-  getDigitalOceanOauthFlow(onCancel) {
-    const oauthFlow = this.$.digitalOceanOauth;
-    oauthFlow.onCancel = onCancel;
-    return oauthFlow;
+  initializeDigitalOceanConnectAccountApp(appSettings, accountRepository, notificationManager) {
```

Comment:
"Initialize" is usually a red flag. Creation should be initialization.
Perhaps you set those in the constructor?

Also, we probably want to get rid of appSettings.